src/components/Book.js

```diff
@@ -1,23 +1,33 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
 
-function Book(props) {
-  const { title, author } = props;
+function Book({ book }) {
+  const { id, title, author } = book;
+
+  const dispatch = useDispatch();
+  const removeHandler = () => {
+    dispatch(removeBook(id));
+  };
 
   return (
     <div>
       <ul>
-        <li>{title}</li>
-        <li>{author}</li>
+        <h3>{title}</h3>
+        <h4>{author}</h4>
       </ul>
-      <button type="button">Remove</button>
+      <button type="button" id={id} onClick={removeHandler}>Remove</button>
```

Comment:
- [ ] I would advise you to implement removing books; adding and removing books must be possible.  It is the way to confirm if the code you implemented is working.
- 
![Capture](https://user-images.githubusercontent.com/63972670/221890492-61955d23-692a-4772-aa8a-1f14edd6792f.PNG)

Reply:
Thank you for the feedback